internet_banking_app/src/main/java/com/gabrielcireap/internet_banking_app/repository/jdbc/AnswerMapper.java

```diff
@@ -0,0 +1,24 @@
+package com.gabrielcireap.internet_banking_app.repository.jdbc;
+
+import com.gabrielcireap.internet_banking_app.entity.Answer;
+import org.springframework.jdbc.core.RowMapper;
+
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Timestamp;
+
+public class AnswerMapper implements RowMapper {
```

Comment:
RowMapper is a generic interface. You should implement RowMapper<Answer> such that your code is type-safe.